src/harness/override-perf-ci-cd-ff.yaml

```diff
@@ -0,0 +1,173 @@
+# Use this override file in addition to override-prod.yaml
+
+ci:
+  ci-manager:
+    autoscaling:
+      enabled: false
+    replicaCount: 4
+    securityContext: {}
+    additionalConfigs:
+      PLAN_CREATOR_SERVICE_EXECUTOR_POOL_CORE_SIZE: "30"
+      PLAN_CREATOR_SERVICE_EXECUTOR_POOL_MAX_SIZE: "100"
+      ASYNC_DELEGATE_RESPONSE_THREAD_COUNT: "20"
+      DISTRIBUTED_LOCK_IMPLEMENTATION: "REDIS"
+    java:
+      memory: "4096m"
+    resources:
+      limits:
+        cpu: 3
+        memory: 6Gi
+      requests:
+        cpu: 3
+        memory: 6Gi
+​platform:
+  # Use the following redis configuration if using redis in-cluster installed by harness.
+  redis:
+    redis:
+      resources:
+        limits:
+          cpu: 8
+          memory: 30Gi
+        requests:
+          cpu: 8
+          memory: 30Gi
+  access-control:
+    autoscaling:
+      enabled: false
+    replicaCount: 3
+    # Update the following configuration if using external mongo
+    # mongoSSL:
+    #     enabled: true
+    # mongoHosts:
+    # - "shard-1"
+    # - "shard-2"
+    java:
+      memory: 3g
+    resources:
+      limits:
+        cpu: 3
+        memory: 6Gi
+      requests:
+        cpu: 3
+        memory: 6Gi
+  pipeline-service:
+    autoscaling:
+      enabled: false
+    replicaCount: 8
+    securityContext: {}
+    additionalConfigs:
+      DISTRIBUTED_LOCK_IMPLEMENTATION: "REDIS"
+      ORCHESTRATION_POOL_CORE_SIZE: "40"
+      ORCHESTRATION_POOL_MAX_SIZE: "300"
+      PMS_SDK_EXECUTION_POOL_CORE_SIZE: "40"
+      PMS_SDK_EXECUTION_POOL_MAX_SIZE: "300"
+      PMS_SDK_ORCHESTRATION_EVENT_POOL_CORE_SIZE: "40"
+      PMS_SDK_ORCHESTRATION_EVENT_POOL_MAX_SIZE: "300"
+      PIPELINE_EXECUTION_POOL_CORE_SIZE: "40"
+      PIPELINE_EXECUTION_POOL_MAX_SIZE: "300"
+      ORCHESTRATION_VISUALIZATION_POOL_CORE_SIZE: "30"
+      ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "50"
```

Comment:
this is additional vs apple

Reply:
ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "50"

Reply:
this is the one which was missing in gist as well. which was causing pipeline service to fail.

Reply:
ok

Reply:
we are using ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "100" in smp-perf cluster, I think it should be 100